blockchain/chain_sync/src/sync.rs

```diff
@@ -229,7 +229,7 @@ where
             self.state.write().await.error(e.to_string());
             return Err(e.into());
         }
-
+        ChainStore::put_tipset(&mut self.chain_store, &*head.clone()).await?;
```

Comment:
isn't this stored otherwise? even if not can you remove the clone? shouldn't need to serialize. maybe `head.as_ref()` is what you want

Reply:
Its not stored otherwise. The bundle of headers above does not include this tipset. put_tipsets also sets the head for the tracker too.

Reply:
it's stored on line 242? Even if you keep, you should not clone the head just to serialize (it is an Arc, but unnecessary clone when you can just take a ref)

Reply:
ah youre right actually, this should be removed

Reply:
fixed